actix-http/README.md

```diff
@@ -13,7 +13,7 @@ Actix http
 ## Example
 
 ```rust
-// see examples/framed_hello.rs for complete list of used crates.
+// see examples/hello-world.rs for complete list of used crates.
```

Comment:
This code snippet currently looks exactly the same as the linked example. Do we need this comment at all?

Reply:
@popzxc I have the same idea.

The only difference is as follows. The example uses `use futures_util::future`.

```diff
- use futures::future;
+ use futures_util::future;
```

Reply:
It has the same effect, I wouldn't mind either way.

Reply:
@JohnTitor 

Thanks. The Cargo.toml depends `futures-util`, not `futures`. Is this the same? The new users of actix-http can easily to copy and past from the Cargo.toml.
https://github.com/actix/actix-web/blob/1596893ef7275a2c6530e09abaec7e2cc2f0bf2c/actix-http/Cargo.toml#L62

Reply:
I mean, it has the same effect so we could just tweak the example on README.

Reply:
I think I should close this PR and someone maintained the README can edit/remove README.md as they like.